docs/getting-started-guides/kubeadm.md

```diff
@@ -202,30 +215,49 @@ In the example above, this was `31869`, but it is a different port for you.
+kubeadm will work fine on ARM, thanks to that sig-cluster-lifecycle has followed the [multi-platform proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/multi-platform.md).
 
+deb-packages are released for ARM and ARM 64-bit, but not RPMs (yet, reach out if there's interest).
 
-## What's next
+Anyway, ARM had some issues when making v1.4, see [#32517](https://github.com/kubernetes/kubernetes/pull/32517) [#33485](https://github.com/kubernetes/kubernetes/pull/33485), [#33117](https://github.com/kubernetes/kubernetes/pull/33117) and [#33376](https://github.com/kubernetes/kubernetes/pull/33376). 
 
-* Learn more about [Kubernetes concepts and kubectl in Kubernetes 101](/docs/user-guide/walkthrough/).
-* Install Kubernetes with [a cloud provider configurations](/docs/getting-started-guides/) to add Load Balancer and Persistent Volume support.
+However, thanks to the PRs above, `kube-apiserver` works on ARM from the `v1.4.1` (`v1.4.1-beta.2` works as well) release, so make sure you're at least using `v1.4.1` when running on ARM 32-bit (this doesn't affect ARM 64-bit at all)
+
+Also, a pod network for ARM is required, and flannel is currently the only one that's providing multi-arch binaries and images.
+Therefore, when initializing the master, run this:
+
+    # kubeadm init --use-kubernetes-version=v1.4.1 --pod-network-cidr=10.244.0.0/16
 
+The `--pod-network-cidr` option is mandatory when using flannel as the pod network provider.
 
-## Cleanup
+And when installing the flannel pod network, run this:
+    # curl -sSL https://raw.githubusercontent.com/luxas/flannel/update-daemonset/Documentation/kube-flannel.yml | sed "s/amd64/arm/g" | kubectl create -f -
+
+The above command will download the flannel pod network manifest, replace the `amd64` references with `arm` ones and create the pod network.
+
+## Turndown
 
 * To uninstall the socks shop, run `kubectl delete -f microservices-demo/deploy/kubernetes/manifests` on the master.
 
 * To undo what `kubeadm` did, simply delete the machines you created for this tutorial, or run the script below and then uninstall the packages.
-  <details>
-     <pre><code>systemctl stop kubelet;
-  docker rm -f $(docker ps -q); mount | grep "/var/lib/kubelet/*" | awk '{print $3}' | xargs umount 1>/dev/null 2>/dev/null;
-  rm -rf /var/lib/kubelet /etc/kubernetes /var/lib/etcd /etc/cni;
-  ip link set cbr0 down; ip link del cbr0;
-  ip link set cni0 down; ip link del cni0;
-  systemctl start kubelet</code></pre>
-  </details> <!-- *syntax-highlighting-hack -->
+    systemctl stop kubelet;
+    docker rm -f $(docker ps -q); mount | grep "/var/lib/kubelet/*" | awk '{print $3}' | xargs umount 1>/dev/null 2>/dev/null;
+    rm -rf /var/lib/kubelet /etc/kubernetes /var/lib/etcd /etc/cni;
+    ip link set cni0 down; ip link del cni0;
+    systemctl start kubelet
+    <!-- *syntax-highlighting-hack -->
```

Comment:
Any ideas what this tag really means? I am not sure I can see any highlighting on this code, and generally shell syntax is just to ambiguous for highlighters...